app/views/booths/_form.html.haml

```diff
@@ -10,6 +10,14 @@
         = f.input :submitter_relationship, input_html: { rows: 5, data: { provide: 'markdown-editable' } }, required: true,
           label: 'Submitter\'s relation',
           hint: 'e.g. employee, comunity manager, etc'
+
+        = f.input :booth_type_id, as: :select,
+          collection: conference.program.booth_types.map {|type| ["#{type.title}", type.id]},
+          include_blank: false, label: 'Type', input_html: { class: 'select-help-toggle' } if conference.program.booth_types.any?
+
+        - conference.program.booth_types.each do |booth_type|
```

Comment:
Can you show me what you expect to show with this? Perhaps it makes sense for the type description to be a hint under the input field?

Reply:
That is what happens currently, whatever type is chosen its description appears as a hint below the field .
![image](https://user-images.githubusercontent.com/26951824/63653985-100dc000-c792-11e9-823b-14d78317bac5.png)